examples/test-app/app.js

```diff
@@ -21,6 +23,8 @@ lookAndFeel.configure(app, {
 journey(app, {
   baseUrl,
   steps: [
+    new Start(),
+    new Entry(),
```

Comment:
i would prefer to define the next step in the proceeding step if possible, otherwise you end up with this large config file just defining all the steps.
 
so all you would have to define is the Start step, that would include the next step and so on

Reply:
You do, this is just a list of all the steps in this journey. I plan to remove this list and replace it with a step finder of some sort, like you could glob for `**/*.step.js` and it would find them all and return an array with each initialised.

Reply:
oh, wait I know what you are talking about now.. 

problem with that is binding the routers to the app. You would need to run through every step and request it's next() then bind every step defined to the app and repeat. Totally doable just a bit magic.

Reply:
glob search would do the job :)

Reply:
it would, I guess I'm just predicting needing to do more wiring up of steps. At the moment they are just a class you initialise but at some point we'll have to sort the "a step and it's template and content are colocated" thing.